### PR TITLE
updating readme and adding notice file

### PR DESCRIPTION
**Pull request by dereklegenzoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ ___
 The UI is a React-based Web App available in the `SearchUI` folder. Navigate to [**SearchUI/README.md**](SearchUI/README.md) for more details on the web app. 
 
 
+## Discliamer
```
Comment (jennifermarsman) [inline README.md:90]: "Disclaimer" is spelled wrong.

File: README.md [hunk 2]
```diff
@@ -87,6 +87,10 @@ ___
...
+
+This repo has been tested with document sets as big as 2,000 documents. If you have a larger dataset or run into any issues with this repo, please open an [issue](https://github.com/Azure-Samples/search-qna-maker-accelerator/issues) and we will get back you.
```
Comment (jennifermarsman) [inline README.md:92]: You are missing a preposition: "with or "to".  "..and we will get back **with** you."

**Review by jennifermarsman — CHANGES_REQUESTED** [review]
Please fix spelling/grammar.

File: README.md (hunk 2 above)
Comment (dereklegenzoff) [reply README.md]: Done, thank you!

File: README.md (hunk 1 above)
Comment (dereklegenzoff) [reply README.md]: Done!

**Review by jennifermarsman — APPROVED** [review]

**Pull request merged**